tests/README.md

```diff
@@ -13,28 +13,25 @@ We need all but the main `backend` service running.  Since multiple `backend` in
     # SETUP ALL IMAGES
     docker-compose up --build
 
-### Install into virtual environment
+## Install into virtual environment
 
 Treeherder requires specific library versions that will likely interfere with what you have installed. It is best to setup a virtual environment to contain the quirks it introduces.
 
 Be sure you are in the `treeherder` main directory
 
-    pip install virtualenv
-    virtualenv .venv                   # IMPORTANT: Notice the dot in the name
+    python -m venv .venv             # IMPORTANT: Notice the dot in the name
```

Comment:
With Python 3 we don't need `virtualenv` as a dependency.

Reply:
in theory, but 3.7.6 still does not work on windows (I see the windows instructions have not changed, so we are good)

Reply:
I actually tested it few weeks ago on Windows that it works. We forgot about the conversation on https://bugzilla.mozilla.org/show_bug.cgi?id=1613127 😄

Reply:
Maybe you tested a docker image of Windows on Mac?  This seems like a file locking issue with Windows on real hardware.